tests/testcase.mk

```diff
@@ -35,17 +35,13 @@ export CDB2_OPTIONS=--cdb2cfg $(CDB2_CONFIG)
 export COMDB2_ROOT=$(TESTDIR)
 export COMDB2_UNITTEST?=0
 
-export COMDB2_EXE?=$(SRCHOME)/comdb2
-export COMDB2AR_EXE?=$(SRCHOME)/comdb2ar
-export CDB2SQL_EXE?=$(SRCHOME)/cdb2sql
-export COPYCOMDB2_EXE?=$(SRCHOME)/db/copycomdb2
-export CDB2_SQLREPLAY_EXE?=$(SRCHOME)/cdb2_sqlreplay
-export PMUX_EXE?=$(SRCHOME)/pmux
-
-ifeq ($(COMDB2MD5SUM),)
-  # record md5sum so we can verify from setup of each individual test
-  export COMDB2MD5SUM:=$(shell md5sum ${COMDB2_EXE} | cut -d ' ' -f1)
-endif
+#also defined in Makefile -- needed here because we can run tests from .test/
+export COMDB2_EXE?=$(TESTDIR)/comdb2
+export COMDB2AR_EXE?=$(TESTDIR)/comdb2ar
+export CDB2SQL_EXE?=$(TESTDIR)/cdb2sql
+export COPYCOMDB2_EXE?=$(TESTDIR)/db/copycomdb2
+export CDB2_SQLREPLAY_EXE?=$(TESTDIR)/cdb2_sqlreplay
```

Comment:
Where is copycomdb2 & cdb2_sqlreplay being copied? executables target above doesn't seem to copy them.

Reply:
Yep, needs fixing.